content/es/tightly-coupled-architectures.md

```diff
@@ -0,0 +1,18 @@
+---
+título: Arquitecturas fuertemente acopladas
+Estado: COMPLETADO
+categoría: Propiedad
+etiquetas: ["fundamental", "arquitectura", "propiedad"]
```

Comment:
This whole section should not be translated. See [this other Spanish localization](https://github.com/cncf/glossary/blob/dev-es/content/es/distributed-systems.md?plain=1#L2-L5) where we keep the words in the original language.